docs/src/main/tut/uri.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: URI handling
+---
+
+## Literals
+
+http4s is a bit more strict with handling URIs than e.g. the [play http client].
+Instead of passing plain `String`s, http4s operates on URIs. You can construct
+literal URI with
+
+```tut:book
+import org.http4s._
+
+val uri = Uri.uri("http://http4s.org")
+```
+
+## Building URIs
+
+Naturally, that's not enough if you want dynamic URIs. There's a few different
+ways to build URIs, you can either use a predefined URI and call methods on it,
+or you could use the URLTemplates.
+
+### URI
+
+Use the methods on the [uri class].
+
+```tut:book
+val docs = uri.withPath("/docs/0.15/")
+```
+
+### URI Template
+
```

Comment:
This section leaves me bewildered what good a URI template is.  Especially because it can be built similarly with a raw URI:

```scala
Uri(authority = Some(Uri.Authority(host = Uri.RegName("http4s.org"))), scheme = Some("http".ci)) / "docs" / "0.15"
```